Platform/ArrowlakeBoardPkg/BoardConfigArlh.py

```diff
@@ -296,6 +298,11 @@ def GetCopyList (self,driver_inf):
                 break
         return copy_list
 
+    # Get IppCrypto inf file based on optimization level.
+    # e.g. if self.ENABLE_CRYPTO_SHA_OPT is 'X64_Y8' then IppCrypto2LibY8.inf will be used.
+    # Each optimization will have its own inf files that will be included in the build.
+    # This function will also copy the required source files to the build folder if
+    # CopyList is defined in the inf file.
     def GetIppCryptoInf(self):
```

Comment:
IPP crypto lib is common module. It is better handle it in common build tool in the PrepareBuildComponentBin.py instead of doing it in every boardConfig script.